interface/forms/CAMOS/content_parser.php

```diff
@@ -13,7 +13,7 @@
  */
 
 require_once("../../../library/api.inc");
-require_once("../vitals/C_FormVitals.class.php");
+require_once($GLOBALS['incdir'] . "/forms/vitals/C_FormVitals.class.php");
```

Comment:
@stephenwaite Should we explicitly include the globals file above api.inc?  api.inc includes the globals.php file but if someone ever removed that implicit dependency, this file will break.  I'm not too turn up about it, just a thought, otherwise this looks good to me.
```
require_once(dirname(__FILE__) . "/../interface/globals.php");
```

Reply:
That is super odd that a library (api.inc) is bringing in globals.php . The script that brings in libraries is responsible for including globals.php ; if allow libraries, then weird things can happen is run library directly from browser or weird things can happen if globals.php is done to early (ie. by a library that should not do it and then skipped by the main script that is supposed to be running with flags set prior to the run). I'm gonna try to remove this from here and api.inc and check out all scripts that include them to ensure nothing breaks.